rhobs/alerting/data_plane/prometheus.gitops_cluster_agent_op_alerts.yaml

```diff
@@ -21,4 +21,5 @@ spec:
           Cluster-Agent Operations in non-complete state are too high. Got: {{ $value }}
         description: >-
           The sum of cluster-agent operations in non-complete state is greater than 10 on cluster {{ $labels.source_cluster }}
+        alert_route_namespace: "gitops-service-argocd"
```

Comment:
GitOps is in the process of being deprecated and the alerting rules will be removed soon (see #179). Consider ignoring them in this PR.

Reply:
I was aware, just not aware how soon will it happen. :)

Reply:
fixed